docs/quickstart.md

```diff
@@ -0,0 +1,84 @@
+# [DRAFT] Quick start
```

Comment:
should we add an entry point on readme page?

Reply:
Once we have the helm chart in MCE and we can skip the hypershift binary parts.

That's my feeling.